commit 87d51bf63b6a96c5187f80f482641005cc67befb
Author: nate zhou <gnuunixchad@outlook.com>
Date:   Wed Feb 11 23:26:54 2026 +0800

    Patch: password-asterisk
    
    Replace the password patch's censort from '.' to '*'.

diff --git a/dmenu.c b/dmenu.c
index 03a0683..675f5de 100644
--- a/dmenu.c
+++ b/dmenu.c
@@ -162,7 +162,7 @@ drawmenu(void)
 	drw_setscheme(drw, scheme[SchemeNorm]);
 	if (passwd) {
 	        censort = ecalloc(1, sizeof(text));
-		memset(censort, '.', strlen(text));
+		memset(censort, '*', strlen(text));
 		drw_text(drw, x, 0, w, bh, lrpad / 2, censort, 0);
 		free(censort);
 	} else drw_text(drw, x, 0, w, bh, lrpad / 2, text, 0);
